src.props

```diff
@@ -46,7 +49,6 @@
 
   <PropertyGroup Condition="'$(Configuration)'=='Release'">
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-		<WarningsAsErrors />
```

Comment:
This is redundant. It is setting no warnings as errors. The line above sets _all_ warnings as errors.